build.sc

```diff
@@ -7,7 +7,7 @@ import coursier.maven.MavenRepository
 // Please retain it.
 // Provide a managed dependency on X if -DXVersion="" is supplied on the command line.
 val defaultVersions = Map(
-  "chisel3" -> "3.3-SNAPSHOT",
+  "chisel3" -> "3.5-SNAPSHOT",
```

Comment:
```suggestion
  "chisel3" -> "3.5.0-RC2",
```

Reply:
nice catch, I forgot to add plugin as well.